arrow_chunk.go

```diff
@@ -17,6 +17,7 @@ type arrowResultChunk struct {
 	rowCount         int
 	uncompressedSize int
 	allocator        memory.Allocator
+	params           map[string]*string
```

Comment:
We could directly hold a lcoation in this new 'params',  to avoid calling the getCurrentLocation function in loops. e.g. in decodeArrowBatch and  decodeArrowChunk.